test/script/reference.t

```diff
@@ -176,7 +176,8 @@
   $ owi script --no-exhaustion reference/proposals/multi-memory/memory_trap1.wast
   $ owi script --no-exhaustion reference/proposals/multi-memory/memory-multi.wast
   $ owi script --no-exhaustion reference/proposals/multi-memory/memory.wast
-  $ owi script --no-exhaustion reference/proposals/multi-memory/simd_memory-multi.wast
+;  $ owi script --no-exhaustion reference/proposals/multi-memory/simd_memory-multi.wast
+; simd instructions are not yet fully supported
```

Comment:
```suggestion
; TODO: simd instructions are not yet fully supported
```

I usually grep "TODO", so I'm simply adding it to be able to find it so it does not go forgotten for ever